src/cpp/parse.cpp

```diff
@@ -4401,6 +4401,16 @@ bool Parser::rClassSpec(typet &spec)
   {
     if(!optAlignas(spec))
       return false;
+
+    if(lex.LookAhead(0)==TOK_GCC_ATTRIBUTE)
```

Comment:
I don't know what this means exactly, but it would probably make sense to have some tests off GCC attributes after struct/class declarations.

Reply:
I'll try to dig up with those tests were: we do have them already, otherwise the change wouldn't have happened :-) - but then it might be rather implicit in that it's drawn in from system headers.